### DISCUSSION TITLE
Quick-Actions same for all profiles?

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15253] [Ideas]
I thought Quick-Actions are different for every profile. E.g. I have "save parking" place in car-profile. But I dont need it in cycle-profile. But when I delete it in the cycle-profile its also deleted in the car-profile? Is this correct? Or a bug?
Can I just turn on/off all (same!) Quick-Actions for every Profile?

**Comment by dmpr0** [comment]
Yes, Quick Actions are profile independent.

**Reply by ThomasMeister123** [reply]
OK, then I switch ist do "Ideas" ;)
In my example "save parking position" (I dont know the correct english name") you see that it is useful to have INDEPENDENT Quickactions

**Reply by OSMJaseB** [reply]
I am interested in this as well. Here is the feature request for this one.
https://github.com/osmandapp/OsmAnd/discussions/11783
